packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1196,6 +1199,11 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
     });
   }
 
+  void _disposeImage([ImageInfo? newInfo]) {
```

Comment:
This is oddly named since it does not dispose the image from the info given to it. Maybe rename to replaceImage?

Reply:
Done